react-training/src/app/screens/Topbar/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
`Topbar` isn't a screen. It's a component for all screens. So move it to `src/app/components`